.catalog-onboard-pipeline.yaml

```diff
@@ -6,7 +6,7 @@ offerings:
     catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
     offering_id: 30a698d4-ab56-4668-bc01-03cd2135d7e4
     variations:
-      - name: standard
+      - name: fully-configurable
         mark_ready: true
         install_type: fullstack
         scc:
```

Comment:
can you opt into this new feature: https://github.com/terraform-ibm-modules/terraform-ibm-scc/pull/262